README.md
# NAME

Dist::Zilla::PluginBundle::Author::CSSON - Dist::Zilla plugin list (deprecated)

# VERSION

Version 0.1103, released 2016-02-18.

# STATUS

Deprecated. See [Dist::Iller::Config::Author::CSSON](https://metacpan.org/pod/Dist::Iller::Config::Author::CSSON) instead.

# SOURCE

Source repository is at [https://github.com/CSSON/p5-Dist-Zilla-PluginBundle-Author-CSSON](https://github.com/CSSON/p5-Dist-Zilla-PluginBundle-Author-CSSON).

# HOMEPAGE

[https://metacpan.org/release/Dist-Zilla-PluginBundle-Author-CSSON](https://metacpan.org/release/Dist-Zilla-PluginBundle-Author-CSSON)

# AUTHOR

Erik Carlsson <info@code301.com>

# COPYRIGHT AND LICENSE

This software is copyright (c) 2016 by Erik Carlsson.

This is free software; you can redistribute it and/or modify it under
the same terms as the Perl 5 programming language system itself.


Changes
Revision history for Dist-Zilla-PluginBundle-CSSON

{{$NEXT}}
 [API Changes]

 [Bug Fixes]

 [Documentation]

 [Enhancements]

 [New Features]

 [Other Changes]

 [Requirements]

 [Stats & Dependency Changes]

 [Testing]

0.1103 2016-02-18T22:13:23Z UTC
 [Other Changes]
 - Deprecated. Will be removed shortly.

 [Stats & Dependency Changes]
 - 9 files changed, 94 insertions(+), 292 deletions(-)
 - (run req) ~ Moose 2.0000 --> 2.1405
 - (run req) ~ MooseX::AttributeShortcuts (any) --> 0.028
 - (run req) ~ Path::Tiny (any) --> 0.072
 - (run req) ~ Types::Standard (any) --> 1.000005
 - (run req) ~ namespace::autoclean (any) --> 0.22
 - (run req) ~ perl 5.010001 --> 5.010000
 - (run req) - Dist::Zilla
 - (run req) - Dist::Zilla::Plugin::CheckChangesHasContent
 - (run req) - Dist::Zilla::Plugin::ConfirmRelease
 - (run req) - Dist::Zilla::Plugin::ContributorsFromGit
 - (run req) - Dist::Zilla::Plugin::CopyFilesFromBuild
 - (run req) - Dist::Zilla::Plugin::ExtraTests
 - (run req) - Dist::Zilla::Plugin::FakeRelease
 - (run req) - Dist::Zilla::Plugin::Git::Check
 - (run req) - Dist::Zilla::Plugin::Git::Commit
 - (run req) - Dist::Zilla::Plugin::Git::GatherDir
 - (run req) - Dist::Zilla::Plugin::Git::Push
 - (run req) - Dist::Zilla::Plugin::Git::Tag
 - (run req) - Dist::Zilla::Plugin::GithubMeta
 - (run req) - Dist::Zilla::Plugin::License
 - (run req) - Dist::Zilla::Plugin::Manifest
 - (run req) - Dist::Zilla::Plugin::ManifestSkip
 - (run req) - Dist::Zilla::Plugin::MetaJSON
 - (run req) - Dist::Zilla::Plugin::MetaNoIndex
 - (run req) - Dist::Zilla::Plugin::MetaYAML
 - (run req) - Dist::Zilla::Plugin::ModuleBuildTiny
 - (run req) - Dist::Zilla::Plugin::NextRelease
 - (run req) - Dist::Zilla::Plugin::NextVersion::Semantic
 - (run req) - Dist::Zilla::Plugin::OurPkgVersion
 - (run req) - Dist::Zilla::Plugin::PodSyntaxTests
 - (run req) - Dist::Zilla::Plugin::PodWeaver
 - (run req) - Dist::Zilla::Plugin::Prereqs::FromCPANfile
 - (run req) - Dist::Zilla::Plugin::PreviousVersion::Changelog
 - (run req) - Dist::Zilla::Plugin::ReadmeAnyFromPod
 - (run req) - Dist::Zilla::Plugin::ReversionOnRelease
 - (run req) - Dist::Zilla::Plugin::ShareDir
 - (run req) - Dist::Zilla::Plugin::Test::EOL
 - (run req) - Dist::Zilla::Plugin::Test::Kwalitee::Extra
 - (run req) - Dist::Zilla::Plugin::Test::NoTabs
 - (run req) - Dist::Zilla::Plugin::TestRelease
 - (run req) - Dist::Zilla::Plugin::TravisYML
 - (run req) - Dist::Zilla::Plugin::UploadToCPAN
 - (run req) - Dist::Zilla::Plugin::UploadToStratopan
 - (run req) - File::chdir
 - (run req) - Git::Wrapper
 - (run req) - IPC::Run
 - (run req) - Pod::Weaver::Config::Assembler
 - (test req) + ExtUtils::MakeMaker (any)
 - (test req) + File::Spec (any)
 - (test req) + IO::Handle (any)
 - (test req) + IPC::Open3 (any)
 - (test req) - Test::NoTabs
 - (test rec) + CPAN::Meta 2.120900
 - (build req) - Module::Build
 - (conf req) + ExtUtils::MakeMaker (any)
 - (conf req) - Module::Build
 - (dev req) + Dist::Zilla 5.000
 - (dev req) + Dist::Zilla::Plugin::Authority 1.009
 - (dev req) + Dist::Zilla::Plugin::BumpVersionAfterRelease::Transitional
   0.007
 - (dev req) + Dist::Zilla::Plugin::ChangeStats::Dependencies::Git 0.0100
 - (dev req) + Dist::Zilla::Plugin::CheckChangesHasContent 0.008
 - (dev req) + Dist::Zilla::Plugin::Clean 0.07
 - (dev req) + Dist::Zilla::Plugin::ConfirmRelease (any)
 - (dev req) + Dist::Zilla::Plugin::CopyFilesFromBuild 0.151680
 - (dev req) + Dist::Zilla::Plugin::Deprecated (any)
 - (dev req) + Dist::Zilla::Plugin::DistIller::MetaGeneratedBy (any)
 - (dev req) + Dist::Zilla::Plugin::ExecDir (any)
 - (dev req) + Dist::Zilla::Plugin::Git 2.036
 - (dev req) + Dist::Zilla::Plugin::Git::Check (any)
 - (dev req) + Dist::Zilla::Plugin::Git::CheckFor::CorrectBranch 0.013
 - (dev req) + Dist::Zilla::Plugin::Git::Commit (any)
 - (dev req) + Dist::Zilla::Plugin::Git::Contributors 0.017
 - (dev req) + Dist::Zilla::Plugin::Git::GatherDir (any)
 - (dev req) + Dist::Zilla::Plugin::Git::Push (any)
 - (dev req) + Dist::Zilla::Plugin::Git::Tag (any)
 - (dev req) + Dist::Zilla::Plugin::GithubMeta 0.54
 - (dev req) + Dist::Zilla::Plugin::InstallRelease 0.008
 - (dev req) + Dist::Zilla::Plugin::License (any)
 - (dev req) + Dist::Zilla::Plugin::MakeMaker (any)
 - (dev req) + Dist::Zilla::Plugin::Manifest (any)
 - (dev req) + Dist::Zilla::Plugin::ManifestSkip (any)
 - (dev req) + Dist::Zilla::Plugin::MetaJSON (any)
 - (dev req) + Dist::Zilla::Plugin::MetaNoIndex (any)
 - (dev req) + Dist::Zilla::Plugin::MetaProvides::Package 2.003001
 - (dev req) + Dist::Zilla::Plugin::MetaYAML (any)
 - (dev req) + Dist::Zilla::Plugin::NextRelease::Grouped 0.0200
 - (dev req) + Dist::Zilla::Plugin::PodSyntaxTests (any)
 - (dev req) + Dist::Zilla::Plugin::PodWeaver 4.006
 - (dev req) + Dist::Zilla::Plugin::Prereqs (any)
 - (dev req) + Dist::Zilla::Plugin::Readme (any)
 - (dev req) + Dist::Zilla::Plugin::ReadmeAnyFromPod 0.150250
 - (dev req) + Dist::Zilla::Plugin::RewriteVersion::Transitional 0.007
 - (dev req) + Dist::Zilla::Plugin::RunExtraTests 0.028
 - (dev req) + Dist::Zilla::Plugin::ShareDir (any)
 - (dev req) + Dist::Zilla::Plugin::Test::Compile 2.054
 - (dev req) + Dist::Zilla::Plugin::Test::EOF 0.0501
 - (dev req) + Dist::Zilla::Plugin::Test::EOL 0.18
 - (dev req) + Dist::Zilla::Plugin::Test::Kwalitee::Extra v0.2.1
 - (dev req) + Dist::Zilla::Plugin::Test::NoTabs 0.15
 - (dev req) + Dist::Zilla::Plugin::Test::ReportPrereqs 0.021
 - (dev req) + Dist::Zilla::Plugin::Test::Version 1.05
 - (dev req) + Dist::Zilla::Plugin::TestRelease (any)
 - (dev req) + Dist::Zilla::Plugin::TravisYML 1.15
 - (dev req) + Dist::Zilla::Plugin::UploadToCPAN (any)
 - (dev req) + Pod::Elemental::Transformer::List (any)
 - (dev req) + Pod::Weaver::Plugin::SingleEncoding (any)
 - (dev req) + Pod::Weaver::Plugin::Transformer (any)
 - (dev req) + Pod::Weaver::PluginBundle::CorePrep (any)
 - (dev req) + Pod::Weaver::Section::Authors (any)
 - (dev req) + Pod::Weaver::Section::Collect (any)

Makefile.PL
# This file was automatically generated by Dist::Zilla::Plugin::MakeMaker v5.043.
use strict;
use warnings;

use 5.010000;

use ExtUtils::MakeMaker;

my %WriteMakefileArgs = (
  "ABSTRACT" => "Dist::Zilla plugin list (deprecated)",
  "AUTHOR" => "Erik Carlsson <info\@code301.com>",
  "CONFIGURE_REQUIRES" => {
    "ExtUtils::MakeMaker" => 0
  },
  "DISTNAME" => "Dist-Zilla-PluginBundle-Author-CSSON",
  "LICENSE" => "perl",
  "MIN_PERL_VERSION" => "5.010000",
  "NAME" => "Dist::Zilla::PluginBundle::Author::CSSON",
  "PREREQ_PM" => {
    "Config::INI" => 0,
    "Dist::Zilla::Role::PluginBundle::Config::Slicer" => 0,
    "Dist::Zilla::Role::PluginBundle::Easy" => 0,
    "Dist::Zilla::Role::PluginBundle::PluginRemover" => 0,
    "List::AllUtils" => 0,
    "Moose" => "2.1405",
    "MooseX::AttributeShortcuts" => "0.028",
    "Path::Tiny" => "0.072",
    "Types::Standard" => "1.000005",
    "namespace::autoclean" => "0.22"
  },
  "TEST_REQUIRES" => {
    "ExtUtils::MakeMaker" => 0,
    "File::Spec" => 0,
    "IO::Handle" => 0,
    "IPC::Open3" => 0,
    "Test::More" => "0.96"
  },
  "VERSION" => "0.1104",
  "test" => {
    "TESTS" => "t/*.t"
  }
);


my %FallbackPrereqs = (
  "Config::INI" => 0,
  "Dist::Zilla::Role::PluginBundle::Config::Slicer" => 0,
  "Dist::Zilla::Role::PluginBundle::Easy" => 0,
  "Dist::Zilla::Role::PluginBundle::PluginRemover" => 0,
  "ExtUtils::MakeMaker" => 0,
  "File::Spec" => 0,
  "IO::Handle" => 0,
  "IPC::Open3" => 0,
  "List::AllUtils" => 0,
  "Moose" => "2.1405",
  "MooseX::AttributeShortcuts" => "0.028",
  "Path::Tiny" => "0.072",
  "Test::More" => "0.96",
  "Types::Standard" => "1.000005",
  "namespace::autoclean" => "0.22"
);


unless ( eval { ExtUtils::MakeMaker->VERSION(6.63_03) } ) {
  delete $WriteMakefileArgs{TEST_REQUIRES};
  delete $WriteMakefileArgs{BUILD_REQUIRES};
  $WriteMakefileArgs{PREREQ_PM} = \%FallbackPrereqs;
}

delete $WriteMakefileArgs{CONFIGURE_REQUIRES}
  unless eval { ExtUtils::MakeMaker->VERSION(6.52) };

WriteMakefile(%WriteMakefileArgs);


Build.PL

# This file was automatically generated by Dist::Zilla::Plugin::ModuleBuild v5.029.
use strict;
use warnings;

use Module::Build 0.3601;


my %module_build_args = (
  "build_requires" => {
    "Module::Build" => "0.3601"
  },
  "configure_requires" => {
    "Module::Build" => "0.3601"
  },
  "dist_abstract" => "Dist::Zilla like Csson",
  "dist_author" => [
    "Erik Carlsson <info\@code301.com>"
  ],
  "dist_name" => "Dist-Zilla-PluginBundle-Author-CSSON",
  "dist_version" => "0.1103",
  "license" => "perl",
  "module_name" => "Dist::Zilla::PluginBundle::Author::CSSON",
  "recommends" => {},
  "recursive_test_files" => 1,
  "requires" => {
    "Config::INI" => 0,
    "Dist::Zilla" => "5.000",
    "Dist::Zilla::Plugin::CheckChangesHasContent" => 0,
    "Dist::Zilla::Plugin::ConfirmRelease" => 0,
    "Dist::Zilla::Plugin::ContributorsFromGit" => 0,
    "Dist::Zilla::Plugin::CopyFilesFromBuild" => 0,
    "Dist::Zilla::Plugin::ExtraTests" => 0,
    "Dist::Zilla::Plugin::FakeRelease" => 0,
    "Dist::Zilla::Plugin::Git::Check" => 0,
    "Dist::Zilla::Plugin::Git::Commit" => 0,
    "Dist::Zilla::Plugin::Git::GatherDir" => 0,
    "Dist::Zilla::Plugin::Git::Push" => 0,
    "Dist::Zilla::Plugin::Git::Tag" => 0,
    "Dist::Zilla::Plugin::GithubMeta" => 0,
    "Dist::Zilla::Plugin::License" => 0,
    "Dist::Zilla::Plugin::Manifest" => 0,
    "Dist::Zilla::Plugin::ManifestSkip" => 0,
    "Dist::Zilla::Plugin::MetaJSON" => 0,
    "Dist::Zilla::Plugin::MetaNoIndex" => 0,
    "Dist::Zilla::Plugin::MetaYAML" => 0,
    "Dist::Zilla::Plugin::ModuleBuildTiny" => 0,
    "Dist::Zilla::Plugin::NextRelease" => 0,
    "Dist::Zilla::Plugin::NextVersion::Semantic" => 0,
    "Dist::Zilla::Plugin::OurPkgVersion" => 0,
    "Dist::Zilla::Plugin::PodSyntaxTests" => 0,
    "Dist::Zilla::Plugin::PodWeaver" => "4.000",
    "Dist::Zilla::Plugin::Prereqs::FromCPANfile" => 0,
    "Dist::Zilla::Plugin::PreviousVersion::Changelog" => 0,
    "Dist::Zilla::Plugin::ReadmeAnyFromPod" => 0,
    "Dist::Zilla::Plugin::ReversionOnRelease" => 0,
    "Dist::Zilla::Plugin::ShareDir" => 0,
    "Dist::Zilla::Plugin::Test::EOL" => 0,
    "Dist::Zilla::Plugin::Test::Kwalitee::Extra" => 0,
    "Dist::Zilla::Plugin::Test::NoTabs" => 0,
    "Dist::Zilla::Plugin::TestRelease" => 0,
    "Dist::Zilla::Plugin::TravisYML" => 0,
    "Dist::Zilla::Plugin::UploadToCPAN" => 0,
    "Dist::Zilla::Plugin::UploadToStratopan" => 0,
    "Dist::Zilla::Role::PluginBundle::Config::Slicer" => 0,
    "Dist::Zilla::Role::PluginBundle::Easy" => 0,
    "Dist::Zilla::Role::PluginBundle::PluginRemover" => 0,
    "File::chdir" => 0,
    "Git::Wrapper" => 0,
    "IPC::Run" => 0,
    "List::AllUtils" => 0,
    "Moose" => "2.0000",
    "MooseX::AttributeShortcuts" => 0,
    "Path::Tiny" => 0,
    "Pod::Weaver::Config::Assembler" => 0,
    "Types::Standard" => 0,
    "namespace::autoclean" => 0,
    "perl" => "5.010001"
  },
  "script_files" => [],
  "share_dir" => {
    "module" => {
      "Dist::Zilla::MintingProfile::Author::CSSON" => "profiles"
    }
  },
  "test_requires" => {
    "Test::More" => "0.96",
    "Test::NoTabs" => 0
  }
);


my %fallback_build_requires = (
  "Module::Build" => "0.3601",
  "Test::More" => "0.96",
  "Test::NoTabs" => 0
);


unless ( eval { Module::Build->VERSION(0.4004) } ) {
  delete $module_build_args{test_requires};
  $module_build_args{build_requires} = \%fallback_build_requires;
}

my $build = Module::Build->new(%module_build_args);

$build->create_build_script;
